pkg/3scale/amp/component/redis.go

```diff
@@ -395,7 +397,7 @@ func (redis *Redis) SystemDeployment() *k8sappsv1.Deployment {
 							ReadinessProbe: &v1.Probe{
 								ProbeHandler: v1.ProbeHandler{
 									Exec: &v1.ExecAction{
```

Comment:
Just curious, why are we removing the `container-endpoint` part from the container command? Does Valkey just not require it?